modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -2273,7 +2300,8 @@ private CompletableFuture<Void> handleChangePendingAssignmentEvent(Entry pending
                                 replicaGrpId,
                                 tables.get(tblId),
```

Comment:
I don't think we can receive null here, because we are using the *recovery revision* and, looking at the code, it seems that all VVs will be inititally completed with this revision (see `IgniteImpl#recoverComponentsStateOnStart`)